net/libtorrent-rasterbar11/Portfile

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           active_variants 1.1
+PortGroup           compiler_blacklist_versions 1.0
+PortGroup           conflicts_build 1.0
+PortGroup           cxx11 1.1
+PortGroup           github 1.0
+
+github.setup        arvidn libtorrent 1_1_13 libtorrent-
+name                libtorrent-rasterbar11
+version             [string map {_ .} ${version}]
+license             BSD
+categories          net
+maintainers         {i0ntempest @i0ntempest} openmaintainer
```

Comment:
your e-mail address needs to be specified as: ```example.org:julesverne``` (otherwise @macports.org) will be appended which is only valid for MacPorts members with commit access.